src/network-layer/default/RelayDefaultNetworkLayer.js

```diff
@@ -121,17 +121,22 @@ class RelayDefaultNetworkLayer {
         }
       }
       init = {
+        ...this._init,
         body: formData,
         method: 'POST',
       };
     } else {
       init = {
+        credentials: 'same-origin',
```

Comment:
Hmm... maybe if you don't trust the server from which you're fetching data? But in that case, you would have had to knowingly set `mode: 'cors'` anyway, so you could just as easily set `credentials: null`.

It's a bit more dangerous of a default, but I could go either way.